SportHub.Business/Services/ArticleService.cs

```diff
@@ -29,7 +30,21 @@ public async Task CreateArticleAsync(Article article)
 			}
 			article.PublishingDate = DateTime.Now;
 
-			await _articleRepository.CreateArticleAsync(article);
+			var imageId = Guid.NewGuid().ToString();
+
+			var fileName = imageId + Path.GetExtension(file.FileName);
+			var parentPath = Directory.GetParent(Directory.GetCurrentDirectory());
+			var filePath = Path.Combine(parentPath.FullName, "SportHub.Database\\Images", fileName);
+			using (var stream = new FileStream(filePath, FileMode.Create))
+			{
+				await file.CopyToAsync(stream);
+			}
+
+			article.ImageId = imageId;
+			image.ImageId = imageId;
+			image.Url = filePath;
```

Comment:
Do we need to store URL on image? - we can calculate value of URL during runtime